client/signup/config/flows-pure.js

```diff
@@ -353,6 +353,15 @@ export function generateFlows( {
 		};
 	}
 
+	if ( isEnabled( 'gutenboarding' ) ) {
+		flows.frankenflow = {
+			steps: [ 'user', 'domains', 'plans' ],
+			description: 'Frankenflow testing flow for Gutenboarding',
```

Comment:
Maybe add a destination is what we need?
```suggestion
			description: 'Frankenflow testing flow for Gutenboarding',
			destination: getEditorDestination,
```